### PR TITLE
Emit persistent event in `verifyPartnerChainOutboxRoot`

### PR DESCRIPTION
**Pull request by ricott1**
- We emit events during `verifyPartnerChainOutboxRoot` in case the verification fails. 
- We extract the verification parts of `beforeCrossChainMessagesExecution` (`verifyCertificateSignature` and `verifyPartnerChainOutboxRoot`) so that result in a failed transaction.

### CODE REVIEW

File: proposals/lip-0053.md [hunk 1]
```diff
@@ -322,11 +322,23 @@ def verifyPartnerChainOutboxRoot(ccu: CCU) -> None:
...
+                topic = []
+            )
             raise Exception("Invalid inclusion proof for inbox update.")
 
     # Verification for empty certificates.
     else:
         if newInboxRoot != channel(partnerchainID).partnerChainOutboxRoot:
+            emitPersistentEvent(
+                module = MODULE_NAME_INTEROPERABILITY,
+                name = EVENT_NAME_INVALID_OUTBOX_ROOT,
```
Comment (AnanyaSshri) [inline proposals/lip-0053.md:338]: As defined earlier in line 1296, "name = EVENT_NAME_INVALID_OUTBOX_ROOT**_VERIFICATION**"

**Review by AnanyaSshri — APPROVED** [review]
Left one comment. Rest seems good to me.

File: proposals/lip-0053.md (hunk 1 above)
Comment (ricott1) [reply proposals/lip-0053.md]: fixed

**Review by gkoumout — APPROVED** [review]

**Review by ishantiw — APPROVED** [review]
Corresponding Lisk SDK issue 
- https://github.com/LiskHQ/lisk-sdk/issues/9111

**Review by janhack — APPROVED** [review]
Editorial approval

**Pull request merged**